nixos/modules/services/network-filesystems/openafs/server.nix

```
@@ -250,6 +250,27 @@ in {
         '';
       };
 
+      privilegedAdministrators = mkOption {
+        default = null;
+        type = types.nullOr (types.listOf types.str);
+        description = ''
+          List of AFS principals that are allowed to carry out privileged
+          administrative operations. If set to null, AFS servers will use the
+          contents of any pre-existing {file}`/etc/openafs/server/UserList`
+          file, instead (see {manpage}`UserList(5)`).
+
+          ::: {.warning}
+          Changing the value of this attribute to anything other than null
+          deletes any pre-existing content from
+          {file}`/etc/openafs/server/UserList` on server startup. Further
+          changes to the list of privileged administrators via the {command}`bos
+          adduser` and {command}`bos removeuser` commands will also be discarded
+          at the next startup.
+          :::
+        '';
+        example = literalExpression ''[ "root.admin" ]'';
```

```suggestion
        example = [ "root.admin" ];
```

This is not a complex value like the ones `literalExpression` is [needed for](https://nixos.org/manual/nixpkgs/stable/#function-library-lib.options.literalExpression).

---

Thank you, I'm still trying to wrap my head around the new documentation best practices. It's been ages since I last contributed an option.